src/rules/players.js

```diff
@@ -0,0 +1,30 @@
+import { Random } from 'boardgame.io/core'
+import { shuffle } from 'lodash'
+
+export function setupPlayers (numPlayers) {
+  const greatPowers = [
+    'England',
+    'Germany',
+    'Russia',
+    'Turkey',
+    'Italy',
+    'France',
+    'Austria'
+  ]
+  // TODO: why is this not shuffling between sessions?
+  const randomizedPowers = Random.Shuffle(greatPowers)
```

Comment:
Just to clarify since I missed this part of the stream -- this doesn't actually get used, correct?  Should we take it out for now?

Reply:
It doesn't get used now, but if I figure out what it's supposed to be doing and how to use it correctly, I'd like to tie it back in.